doc/node.1

```diff
@@ -182,26 +182,34 @@ Specify ICU data load path. (overrides \fBNODE_ICU_DATA\fR)
 \',\'\-separated list of core modules that should print debug information.
 
 .TP
-.BR NODE_PATH =\fIpath\fR[:\fI...\fR]
-\':\'\-separated list of directories prefixed to the module search path.
+.BR NODE_DISABLE_COLORS =\fI1\fR
```

Comment:
Ah I see what you mean. I still think it'be more consistent to still highlight the `1`, maybe the description below as well.